e2e/framework/provisioning/runner_linux.go

```diff
@@ -1,25 +1,57 @@
 package provisioning
```

Comment:
Thanks for taking another crack at this file... I admit it was fairly under-developed. 😀